src/ape/contracts/base.py

```diff
@@ -369,6 +374,7 @@ def range(
 
         start_block = None
         stop_block = None
+        addresses = [self.contract.address, *(extra_addresses or [])]
```

Comment:
a bit cleaner way to write this
```suggestion
        addresses = [self.contract.address] + (extra_addresses or [])
```